src/vacancies/vacancies-service.js

```diff
@@ -0,0 +1,21 @@
+const VacancyService = {
+    getAllVacancies(knex, id) {
+        return knex.from('vacancies').select("*").where({project_id: id}).orderBy('id')
+    },
+    insertVacancy(knex, role) {
+        return knex.insert(role)
+            .into('vacancies')
+            .returning("*")
+            .then(rows => 
+                rows[0]
+            )
+    },
+    updateVacancy(knex, id, vacancy) {
+        return knex('vacancies').where({id}).update(vacancy)
+    },
+    deleteVacancy(knex, id) {
+        return knex('vacancies').where({id}).delete()
```

Comment:
```suggestion
        return knex('vacancies').where({ id }).del()
```

Has delete been tested? AFAIK delete is a reserved word in javascript, so knex uses .del() instead
More info on this at: [https://knexjs.org/#Builder-del%20/%20delete](url)